pvm.py

```diff
@@ -37,18 +38,48 @@ def _init_logger():
 media = ""
 IS_FILE_SET = False
 
+'''
+UDP command example:
+/PVM HH MM SS <command> <value>
+'''
+
 def parse_commands(*args):
 	global media
 	global VIDEO_PATH
 	global IS_FILE_SET
-	command = args[1]
-	_logger.info("Received command: %s", command)
-	if len(args)>2:
-		value = args[2]
-		_logger.info("Received command: %s", str(value))
-		pass
-	# TODO: Create another python file to control two display
+	
+	# Parse time and add 3 seconds
+	hh, mm, ss = args[1], args[2], args[3]
+	time_str = str(hh) + ":" + str(mm) + ":" + str(ss)
+	next_time = datetime.strptime(time_str, "%H:%M:%S") + timedelta(seconds=3)
+
+	# Get command
+	command= args[4]
+
+	# If scheduled time is behind current time, return.
+	if next_time.time() <= datetime.now().time():
+		_logger.info("Command: %s failed because scheduled time(%s) is behind current time.", command, time_str)
+		return
+
+	# Log command and execute time.
+	_logger.info("Command: %s, execute time: %s.", command, next_time.time())
+	
+	# Wait until current time is equal to next_time
+	while True:
+		now = datetime.now()
+		now = now.replace(microsecond=0)
+		if now.time() >= next_time.time():
+			break
+		sleep(0.005)
+	
+	message = "command: " + command
+	if len(args) == 6:
+		value = args[5]
+		message += ", value: " + value
+	_logger.info("Start execute %s", message)
```

Comment:
I guess it would make more sense to print the log entry after the command is successfully executed, or something else if it fails. but I'm not going to make that halt your development strat here.

Reply:
Agreed, remove redundant log